tools/labs/templates/kernel_api/3-memory/memory.c

```diff
@@ -38,14 +38,14 @@ static int memory_init(void)
 {
```

Comment:
Usually we should create one patch per change (but this is small so we can go with it for now). 

Also, please use `tools: labs: Fix typos in kernel_api` for subject.

Reply:
I changed the subject. Thanks for review!

Reply:
@lolteanu you also need to change the title of your commit.

Just run `git commit --amend` change the title then force push the patch and it will update the Pull Request.

Reply:
It took a while for the changes to arrive here :)